services/weblate/weblate-entities.tester.js

```diff
@@ -1,10 +1,6 @@
-import { ServiceTester } from '../tester.js'
+import { createServiceTester } from '../tester.js'
 import { isMetric } from '../test-validators.js'
-export const t = new ServiceTester({
-  id: 'WeblateEntity',
-  title: 'Weblate Entity',
-  pathPrefix: '/weblate',
-})
+export const t = await createServiceTester()
 
 t.create('Components')
   .get('/components.json?server=https://hosted.weblate.org')
```

Comment:
In cases where we have an optional query param used to override a default instance, we like to have test cases that validate both the default (i.e. no query param provided in the test used in the route) and at least one test that validates the query param is usable by supplying the param, _preferrably_ with a different instance than the default when possible (though not necessary)